Scripts/Python/xYeeshaPages.py

```diff
@@ -166,22 +167,24 @@ def OnGUINotify(self,id,control,event):
                 
                 PtDebugPrint ("xYeeshaPages.py: The previous value of the SDL variable %s is %s" % ("YeeshaPage" + str(PageNumber.value), YeeshaPageVar.getInt()))
 
+                if StartInCleft() and not IsCleftSolved():
+                    PtFindSceneobject("YeeshaPageButton","GUI").runAttachedResponder(1)
+                else:
+                    PtFindSceneobject("YeeshaPageButton","GUI").runAttachedResponder(0)
+
                 if YeeshaPageVar.getInt() != 0: 
                     PtDebugPrint ("xYeeshaPages.py: You've already found Yeesha Page #%s. Move along. Move along." % (PageNumber.value))
                     return
                     
                 else:
-                    start = vault.findChronicleEntry("StartPathChosen")
-                    entryCleft = vault.findChronicleEntry("CleftSolved")
-                    if start.chronicleGetValue() == "cleft" and entryCleft is None:
-                        return
```

Comment:
It looks like you're now allowing folks who do not yet have their Relto book to collect yeesha pages?

Reply:
Yes, I changed the sound so it plays a page turn sound when you pick them up as well.
Just makes for a better user experience, being able to pick them up no matter what.